intake_stac/catalog.py

```diff
@@ -162,13 +162,53 @@ def _get_metadata(self, **kwargs):
         metadata.update(kwargs)
         return metadata
 
+    def stack_bands(self, bands):
+        """
+        Stack the listed bands over the ``band`` dimension.
+
+        Parameters
+        ----------
+        bands : list of strings representing the different bands
+        (e.g. ['B1', B2']).
+
+        Returns
+        -------
+        Catalog entry containing listed bands with ``band`` as a dimension
+        and coordinate.
+
+        """
+        item = {'concat_dim': 'band', 'urlpath': []}
+        titles = []
+        for band in bands:
+            value = self._stac_obj.assets[band]
```

Comment:
Do you think its worth catching the potential key error here and returning an informative error message?

Reply:
Sure we could catch it and return a list of valid options.